### DISCUSSION TITLE
The AI for document recognition is really bad.

### DISCUSSION THREAD

**Discussion by Dichtrich** [community/community#121125] [GitHub Education]
### Hi everyone,

Hello, it is impossible to submit documents for the Student pack if you don't yet have a valid ID card.
Firstly, you ask people to translate the documents to English, which with today's technology you don't need that at all.
I have a , dated certified and signed letter that i am a student of a university, the name in the letter is exactly as appears in the billing information, the school name and dates... ALSO there.
Yet i keep getting _"Your document does not appear to include a date demonstrating current academic affiliation"_ 
My admission letter has every proof needed to be accepted in the developer pack. but this AI system is useless,
and worst thing is that there is no way to get personal validation if everything else fails.... i spent 4hs trying to accommodate the document to match Github's requirements and still nothing, is not that I'm lazy and want to avoid preparing it... but not having someone who i can send the documents for a Human review is crazy... 
What else can i do???

**Comment by samus-aran** [comment]
Hi @Dichtrich 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

> it is impossible to submit documents for the Student pack if you don't yet have a valid ID card.

We do accept other forms of evidence, but if the school specifically distributes Student ID then it's the preferred. 

- [What types of evidence does the Student Developer Pack application accept?](https://github.com/orgs/community/discussions/111352#what-types-of-evidence-does-the-student-developer-pack-application-accept)
- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

> Firstly, you ask people to translate the documents to English, which with today's technology you don't need that at all.

The verification process does translates a huge number of languages but not all. So we suggest applicants to translate so they don't lose the opportunity to apply because we don't automatically support their language.

Here are a few tips regarding your rejection reasons:

- You may take a photo of your academic proof directly with your device camera, _instead of using an upload._
- Ensure the text is legible

**Comment by Dichtrich** [comment]
Hello @samus-aran 
i did all that,  
checked the help hundreds of times
tried bolding the details like, date or my name so whatever is trying to capture the details see them better.... 
translated the document myself... spanish to english nothing too crazy. 
the date was mentioend in the text, i added it as a sort of header, since i dont know what "academic format" means, but it  Aril 25th 2024 was really legible there... 
nothing works.

i would love to show you the struggle live if you want. 
also, why is it have to be a camera? when uploading the doc directly is WAY more legible. ? is that a security thing?
